packages/shared/src/libs/modules/http/libs/enums/http-code.enum.ts

```diff
@@ -4,6 +4,7 @@ const HTTPCode = {
 	INTERNAL_SERVER_ERROR: 500,
 	NOT_FOUND: 404,
 	OK: 200,
+	REQUEST_LIMIT: 429,
```

Comment:
let's use its standard name
```suggestion
	TOO_MANY_REQUESTS: 429,
```